src/query_results.js

```diff
@@ -204,149 +204,148 @@ module.exports = class QueryResult {
    * At a high level, this method does the following:
    * 1. Create sets of records such that:
    *    - each set has one record per QEdge and
-   *    - each record in a set has the same primaryID as its neighbor(s) at the same QNode.
-   *    We're calling each set a preresult, but this could be alternatively named atomicResult
-   *    or unconsolidatedResult. There can be one or more preresults per query result.
+   *    - each record in a set has the same primaryCurie as its neighbor(s) at the same QNode.
+   *    We're calling each set a queryGraphSolution.
    * 2. Group the sets by result ID. There will be one group per query result.
-   * 3. Consolidate each group. We're calling each consolidated group a consolidatedPreresult.
-   *    Each consolidatedPreresult becomes a query result.
-   * 4. Format consolidatedPreresults to match the translator standard for query results
+   * 3. Consolidate each group. We're calling each consolidated group a consolidatedResults.
```

Comment:
This should be singular here. What do you think of `consolidatedResult` vs. `consolidatedSolution`?

Reply:
Agreed, we're now using `queryGraphSolution` and `consolidatedSolution`, with no short-forms to avoid mistaking the two.